packages/ui/Scrollable/ScrollableNative.js

```diff
@@ -894,40 +908,6 @@ class ScrollableBaseNative extends Component {
 		this.startHidingThumb();
 	}
 
-	scrollOnScroll = (left, top) => {
-		const
-			bounds = this.getScrollBounds(),
-			canScrollHorizontally = this.canScrollHorizontally(bounds);
-
-		if (!this.scrolling) {
-			this.scrollStartOnScroll();
-		}
-
-		if (this.props.rtl && canScrollHorizontally) {
-			/* FIXME: RTL / this calculation only works for Chrome */
-			left = bounds.maxLeft - left;
-		}
-
-		let
-			dirHorizontal = 0,
-			dirVertical = 0;
-
-		if (left !== this.scrollLeft) {
-			dirHorizontal = Math.sign(left - this.scrollLeft);
-			this.setScrollLeft(left);
-		}
-		if (top !== this.scrollTop) {
-			dirVertical = Math.sign(top - this.scrollTop);
-			this.setScrollTop(top);
-		}
-
-		if (this.childRefCurrent.didScroll) {
-			this.childRefCurrent.didScroll(this.scrollLeft, this.scrollTop, dirHorizontal, dirVertical);
```

Comment:
@0x64 : can you confirm if `dirHorizontal`, `dirVertical` were crufts left over from #2125?

Reply:
@viodragon2 ✔️ confirmed.